core/src/wallet/ripple/api_impl/RippleLikeTransactionApi.cpp

```diff
@@ -102,6 +102,10 @@ namespace ledger {
 
             _sequence = std::make_shared<api::BigIntImpl>(tx.sequence);
 
+            if (tx.destinationTag.nonEmpty()) {
+              _destinationTag = (int64_t)tx.destinationTag.getValue();
```

Comment:
I suggest here to use a proper cast (e.g `static_cast<int64_t>(tx.destinationTag.getValue())`)

Reply:
Thanks for the suggestion. I didn't know this `static_cast` stuff. Updated my PR!